rsdroid-instrumented/src/androidTest/java/net/ankiweb/rsdroid/BackendIntegrationTests.java

```diff
@@ -42,7 +42,7 @@ public class BackendIntegrationTests extends InstrumentedTest {
 
     /** Ensure that the database can't be locked */
     @Rule
-    public Timeout timeout = new Timeout(3, TimeUnit.SECONDS);
+    public Timeout timeout = new Timeout(10, TimeUnit.SECONDS);
```

Comment:
Now that I am running these tests more regularly, it still sometimes failed, even at 30 so I widened it to 60
It's really just a "DB totally locked / total hang" fail-safe that should not be hit, so expanding duration until it does not
flake seems fine even out from previous 3 to 60. This works for me anyway

```suggestion
    public Timeout timeout = new Timeout(60, TimeUnit.SECONDS);
```